packages/superset-ui-chart-controls/src/sections/forecastInterval.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { legacyValidateInteger, legacyValidateNumber, t } from '@superset-ui/core';
+import { ControlPanelSectionConfig } from '../types';
+
+export const forecastEnabled = false;
+export const forecastInterval = 0.8;
+export const forecastPeriods = 10;
+export const forecastSeasonalityDaily = null;
+export const forecastSeasonalityWeekly = null;
+export const forecastSeasonalityYearly = null;
```

Comment:
Can we export those params as a single object? Like `const FORECAST_DEFAULT_DATA`?